modules/stanford_profile_styles/lib/scss/stanford_page.scss

```diff
@@ -0,0 +1,44 @@
+@charset 'UTF-8';
+@import './config';
+
+//
+// Custom Styles For Stanford Page Content Type.
+//
+.su-page-components {
+  > .ptype-stanford-card,
+  > .ptype-stanford-wysiwyg {
+    max-width: 980px;
+    margin: auto auto;
+  }
+}
+
+// Full width layout
+.ds-1col {
```

Comment:
@cjwest, the webpack integration and compilation is working well now. You may finish what you were working on.